examples/amp-subscriptions-google/amp-subscriptions-metering-gaa.amp.html

```diff
@@ -0,0 +1,195 @@
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-ms-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @-o-keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+      @keyframes -amp-start {
+        from {
+          visibility: hidden;
+        }
+        to {
+          visibility: visible;
+        }
+      }
+    </style>
+    <noscript
+      ><style amp-boilerplate>
+        body {
+          -webkit-animation: none;
+          -moz-animation: none;
+          -ms-animation: none;
+          animation: none;
+        }
+      </style></noscript
+    >
+    <script
+      async
+      custom-element="amp-subscriptions"
+      src="https://cdn.ampproject.org/v0/amp-subscriptions-0.1.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-subscriptions-google"
+      src="https://cdn.ampproject.org/v0/amp-subscriptions-google-0.1.js"
+    ></script>
+    <script
+      async
+      custom-template="amp-mustache"
+      src="https://cdn.ampproject.org/v0/amp-mustache-latest.js"
+    ></script>
+    <script
+      async
+      custom-element="amp-analytics"
+      src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"
+    ></script>
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script type="application/ld+json">
+      {
+        "@context": "http://schema.org",
+        "@type": "NewsArticle",
+        "isAccessibleForFree": "False",
+        "publisher": {
+          "name": "The Scenic - USA",
+          "@type": "Organization",
+          "@id": "scenic-2017.appspot.com",
+          "logo": {
+            "@type": "ImageObject",
+            "url": "https://scenic-2017.appspot.com/icons/icon-2x.png"
+          }
+        },
+        "isPartOf": {
+          "@type": ["CreativeWork", "Product"],
+          "name": "Product A",
+          "productID": "scenic-2017.appspot.com:news"
+        }
+      }
+    </script>
+    <script type="application/json" id="amp-subscriptions">
+      {
+        "services": [
+          {
+            "authorizationUrl": "//localhost:8000/subscription/0/entitlements?rid=READER_ID",
+            "pingbackUrl": "//localhost:8000/subscription/pingback",
+            "actions": {
+              "login": "//localhost:8000/subscription/login",
+              "subscribe": "//localhost:8000/subscription/subscribe"
+            }
+          },
+          {
+            "serviceId": "subscribe.google.com",
+            "enableMetering": true,
+            "googleSignInHelperUrl": "http://localhost:8000/examples/amp-subscriptions-google/amp-subscriptions-metering-registration-shim.html",
+            "meteringRegistrationUrl": "http://localhost:8000/subscription/register"
```

Comment:
@ed-bird The SwG service config has three new params
- `enableMetering` - Enables Showcase metering
- `googleSignInHelperUrl` - Points to the publisher's GSI button shim
- `meteringRegistrationUrl` - Points to the publisher's account registration URL

Reply:
nit: can we change the the url to "amp-subscriptions-metering-registration-widget.html" instead of "shim" in the examples?

Reply:
Do we want to say `enableExtendedAccess` and `extendedAccessRegistrationUrl` ?

Reply:
- "shim" has been changed to "widget"
- "meteringRegistrationUrl" has been changed to "extendedAccessRegistrationUrl"

I think keeping "enableMetering" general would be best. I updated the `amp-subscriptions` extension so it treats platforms differently if they have a "enableMetering" `true` setting. Someone might want to make another platform in the future that has a different system for offering metering. What do you think?

Reply:
+1 for keeping it as metering

Reply:
Awesome, thanks!